src/caret_analyze/plot/visualize_lib/bokeh/timeseries.py

```diff
@@ -66,20 +67,51 @@ def create_figure(self) -> Figure:
         # Apply xaxis offset
         records_range = Range([to.to_records() for to in target_objects])
         frame_min, frame_max = records_range.get_range()
-        if self._xaxis_type == 'system_time':
+        converter: Optional[ClockConverter] = None
+        if self._xaxis_type == 'sim_time':
+            # TODO: refactor
+            # get converter
+            if isinstance(target_objects[0], Communication):
+                for comm in target_objects:
+                    assert isinstance(comm, Communication)
+                    if comm.callback_subscription:
+                        converter_cb = comm.callback_subscription
+                        converter = converter_cb._provider.get_sim_time_converter()
+                        break
+            else:
+                converter = target_objects[0]._provider.get_sim_time_converter()
+        if converter:
+            frame_min_convert = converter.convert(frame_min)
+            frame_max_convert = converter.convert(frame_max)
+            x_range_name = 'x_plot_axis'
+            apply_x_axis_offset(fig, frame_min_convert, frame_max_convert, x_range_name)
+        elif self._xaxis_type == 'system_time':
             apply_x_axis_offset(fig, frame_min, frame_max)
 
         # Draw lines
         color_selector = ColorSelectorFactory.create_instance(coloring_rule='unique')
         legend_manager = LegendManager()
-        line_source = LineSource(legend_manager, target_objects[0], frame_min, self._xaxis_type)
+        if converter:
+            line_source = \
+                LineSource(legend_manager, target_objects[0], frame_min_convert, self._xaxis_type)
+        else:
+            line_source = \
+                LineSource(legend_manager, target_objects[0], frame_min, self._xaxis_type)
```

Comment:
```suggestion
        line_source = LineSource(legend_manager, target_objects[0], frame_min, self._xaxis_type)
```

Reply:
I accept this proposal.

Reply:
From the difference between the return values of `records_range.get_range()` and `converter.convert()`, it is better to have another variable for `flame_min/max` and `flame_min/max_contert`.

Hence, there is no need to apply the above suggestion.